### DISCUSSION TITLE
Change default content of html

### DISCUSSION THREAD

**Discussion by edgar-koster** [ueberdosis/tiptap#938] [General Questions] [answered]
The default content seems to be a heading with a paragraph. Is there a way I can change it to be 'empty' or at least only a paragraph, users in my case don't like this and keep thinking the html is incorrect (they even request a button to view the code)

**Comment by hanspagel** [comment, marked as answer]
Sure! You can pass `content: null` or `content: '<p></p>'` to the editor. tiptap will render an empty paragraph in both cases. Hope that helps!

**Reply by edgar-koster** [reply]
This helps thanks. There is still a sub-effect, but I think that is not tiptap itself todo with the first line always switching to a header. I'll do some further checking on that first.

**Reply by hanspagel** [reply]
Sounds like you started with that example: https://github.com/ueberdosis/tiptap/tree/main/examples/Components/Routes/Title

Which defindes in https://github.com/ueberdosis/tiptap/blob/main/examples/Components/Routes/Title/Doc.js#L7 that a document always starts with a title.
